BEIMA.Client/cypress/integration/DeviceTypesPage.spec.js

```diff
@@ -12,9 +12,9 @@ describe("Verify the list has items present", function () {
   it('Count children of item list (see comments)', function (){
     cy.visit('http://localhost:3000/deviceTypes')
 
-    //get all children of the itemlist with class ItemList_item__2BOfJ
-    //there should be at least 1 item in the list
-    cy.get('[id=itemList]').find('.ItemList_item__2BOfJ').its('length').should('be.gt', 0);
-
+    //get all elements on the page with a class that starts with "ItemList_item"
+    //if there's at least one device in the DB it will pass since 2 elements with
+    //the expected class are created for each device listing
+    cy.get('[class^="ItemList_item"]').its('length').should('be.gt', 2);
```

Comment:
Is this meant to be greater than or greater than/equal to 2.

Reply:
With 1 device type in the DB there's a minimum of 3 items that will match the selector:
![image](https://user-images.githubusercontent.com/46760776/161160040-c74bafa3-11dd-4570-b39e-fcbee0149fc0.png)

So I set the test to look for more than min-1 so if there's 1 device the count will be greater than 2. As I was verifying I think it would be more understandable at >= 3 so it looks for the absolute minimum instead

- [ ] change test to >=3 rather than >2

Reply:
I was more going off of the comment, which seemed to imply that it was >= 2, but I could just be misunderstanding